src/container/srv_container.c

```diff
@@ -3144,6 +3149,26 @@ enum_cont_cb(daos_handle_t ih, d_iov_t *key, d_iov_t *val, void *varg)
 	cinfo = &ap->conts[ap->ncont];
 	ap->ncont++;
 	uuid_copy(cinfo->pci_uuid, cont_uuid);
+
+	/* Get the label property. FIXME: cont_lookup no need to search
+	 * in cs_conts, since we're iterating that KVS already.
+	 * Isn't val the container properties KVS? Can it be used directly?
```

Comment:
Yes. The value should be a rdb_oid_t pointing to the container property KVS. rdb API-wise, it's tricky to support this, for we don't have a public KVS descriptor.